### DISCUSSION TITLE
no-tick-viewdistance for afk players

### DISCUSSION THREAD

**Discussion by fredagsguf** [PurpurMC/Purpur#727] [__Dismissed__] [answered]
I would love to see a configurable no-tick view distance for players that are afk.

Not sure what else to provide, it would increase performance for servers with many afk players 😄

**Comment by hirata-estate** [comment]
commenting to bump this up, cuz this'll be amazing

**Comment by YouHaveTrouble** [comment, marked as answer]
How would that affect performance?

Server sends non-simulated chunks *once*, so why bother changing that when player does not move?
